### PR TITLE
COMMERCE-11492 Drop DDM 5/5 - Clean up #4141

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/COMMERCE-11492

Re-sent from https://github.com/liferay-commerce/liferay-portal/pull/4127

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by baharturk** [comment]
There are conflicts on the PR, could you fix them? Thanks!

@andrea-ale-sbarra @alessiorendina

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/registry/CommerceProductServiceUpgradeStepRegistrator.java [hunk 1]
```diff
@@ -389,6 +389,15 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 			new com.liferay.commerce.product.internal.upgrade.v5_7_0.
 				CPDefinitionLinkUpgradeProcess(_assetEntryLocalService));
 
+		registry.register(
+			"5.7.0", "5.8.0",
+			UpgradeProcessFactory.alterColumnName(
+				"CPDefinitionOptionRel", "DDMFormFieldTypeName",
```
[the remark this answers is not in the document]
Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/registry/CommerceProductServiceUpgradeStepRegistrator.java:395]: Thank you @andrea-ale-sbarra

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
Sent here: https://github.com/liferay-commerce/liferay-portal/pull/4198

**Pull request closed without merging**